sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/stack_ensemble_settings.py

```diff
@@ -0,0 +1,71 @@
+from typing import Any
+
+from azure.ai.ml._restclient.v2022_10_01_preview.models import (
+    StackEnsembleSettings as RestStackEnsembleSettings,
+    StackMetaLearnerType
+)
+
+from azure.ai.ml.entities._mixins import RestTranslatableMixin
+
+class StackEnsembleSettings(RestTranslatableMixin):
+    """Advances setting to customize StackEnsemble run.
+    """
+
+    def __init__(
+        self,
+        *,
+        stack_meta_learner_k_wargs: Any = None,
```

Comment:
nit: kwargs

Reply:
maybe I can leave this one as it is for now to align with rest contracts... ?